src/index.js

```diff
@@ -0,0 +1,57 @@
+import './style.css';
+// import render from './populateui.js';
+// import AddToDoList from './list.js';
```

Comment:
- [ ] Please do not leave unused code as a comment in your codebase. this will make your code much noisy, It is always a good practice to keep your code clean from such comments. 😉 👍🏻